packages/components/badge/src/badge.ts

```diff
@@ -25,14 +26,30 @@ export class Badge extends LitElement {
   /** @internal */
   static override styles: CSSResultGroup = styles;
 
-  /** The emphasis of the badge; defaults to 'subtle'. */
-  @property({ reflect: true }) emphasis: BadgeEmphasis = 'subtle';
+  /**
+   * The color of the badge.
+   * @default grey
+   */
+  @property({ reflect: true }) color?: BadgeColor;
 
-  /** The size of the badge component. */
-  @property({ reflect: true }) size: BadgeSize = 'md';
+  /**
+   * The emphasis of the badge.
+   * @default subtle
+   */
+  @property({ reflect: true }) emphasis?: BadgeEmphasis;
 
-  /** The variant of the badge. */
-  @property({ reflect: true }) variant: BadgeVariant = 'neutral';
+  /**
+   * The size of the badge component.
+   * @default md
+   */
+  @property({ reflect: true }) size?: BadgeSize;
+
+  /**
+   * The variant of the badge.
```

Comment:
`deprecated` is not being rendered on the website.


<img width="919" alt="image" src="https://github.com/user-attachments/assets/1dd5d1b7-4b02-4592-a027-7f6f3e194392" />



Maybe it's worth to add this information to the description as well? So sth like:
```suggestion
   * The variant of the badge. This property is deprecated. Use the color property instead.
```
?